### DISCUSSION TITLE
A/UX 3 support?

### DISCUSSION THREAD

**Discussion by rofore** [autc04/Retro68#248] [Ideas]
Early versions of GCC supported A/UX but these can no longer be compiled to run on modern machines. Would there be any chance to use Retro68 to produce A/UX command line binaries, allowing us to cross-compile modern software (like busybox) for A/UX?

**Comment by jduerstock** [comment]
Strictly a personal opinion, but it might be easier to look at the patches used to obsolete the file format than it is to fit things back in Retro68.
